cmd/csi-provisioner/csi-provisioner.go

```diff
@@ -59,12 +59,32 @@ var (
 	workerThreads        = flag.Uint("worker-threads", 100, "Number of provisioner worker threads, in other words nr. of simultaneous CSI calls.")
 	operationTimeout     = flag.Duration("timeout", 10*time.Second, "Timeout for waiting for creation or deletion of a volume")
 	provisioner          = flag.String("provisioner", "", "This option is deprecated")
+	inTreePluginName     = flag.String("supersedes-in-tree-plugin-name", "", "An in-tree plugin that is superseded by this CSI plugin")
```

Comment:
I would appreciate "migration" somewhere in the parameter name and description. `--migrated-plugin-name`?